### PR TITLE
added README and whiteboard

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x ] Picture of whiteboard

File: javascript/code-challenges/graph/getEdge/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Big O
+
+Time - O(n)
+Space - O(n)
+
+## Whiteboard
+
+![Whiteboard](whiteboard.jpg)
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/graph/getEdge/README.md:24]: Once you have a chance to get code and tests written, I will regrade for you!
